elements-fun/src/wally.rs

```diff
@@ -353,7 +352,11 @@ pub fn asset_final_vbf(
     final_vbf
 }
 
-pub fn asset_value_commitment(value: u64, vbf: [u8; 32], generator: Asset) -> Value {
+pub fn asset_value_commitment(
+    value: u64,
+    vbf: [u8; 32],
+    generator: AssetCommitment,
+) -> ValueCommitment {
```

Comment:
:thinking: It's interesting that `libwally` calls this `asset_value_commitment` but we choose `ValueCommitment` as the type.

Reply:
We could change the name of the function.